reference/3.0/PSDiagnostics/Get-LogProperties.md

```diff
@@ -3,38 +3,53 @@ external help file: PSDiagnostics-help.xml
 Module Name: PSDiagnostics
 online version:
 schema: 2.0.0
+ms.date:  11/27/2018
```

Comment:
This wasn't added to each file updated.  Should it be?

Reply:
It should have been copied to all versions. I will double check.